src/lib/DataSourceOrigin.js

```diff
@@ -79,7 +82,7 @@ var DataSourceOrigin = DataSourceBase.extend('DataSourceOrigin',  {
                 schema[i] = { name: fields[i] };
             }
         }
-
+        schema = setInternalColSchema.call(this, schema);
```

Comment:
What does this accomplish?

Reply:
Just reverting the breaking change back to [__your__ code](https://github.com/fin-hypergrid/core/commit/c540b45ace615d757147861a48c26460493455b1#diff-4a3b59fc30ab47d3eee9791d7bc35230R335)!

Reply:
ok